src/main/java/com/code_intelligence/jazzer/mutation/engine/SeededPseudoRandom.java

```diff
@@ -118,6 +118,18 @@ public long closedRange(long lowerInclusive, long upperInclusive) {
     }
   }
 
+  public float closedRange(float lowerInclusive, float upperInclusive) {
+    require(lowerInclusive <= upperInclusive);
+    return (float) random.nextDouble(-Float.MAX_VALUE, Float.MAX_VALUE);
```

Comment:
This is rather confusing: The function ignores its arguments. You could either get rid of the arguments or honor them.

Reply:
Moved the code to the mutator.